chrome-extension/service.js

```diff
@@ -0,0 +1,37 @@
+/**  
+ * @param {String} linkUrl, to be analyzed
+ * @returns {String} sentiment analysis message
+ */
+async function getSentimentAnaylsisMsg(linkUrl) {
+  // TODO use regular expression 
+  const videoId = linkUrl.split('v=')[1];
+  return fetch(`https://step70-2020.appspot.com/sentiment?video-id=${videoId}`)
+    .then(response => {
+      if (response.ok) {
+        return response.json();
+      } else {
+        throw new Error(response.statusText);
+      }
+    })
+    .then(videoAnalysis => {
+      if (videoAnalysis.scoreAvailable) {
+        return videoAnalysis.score.toString() + ' out of ±1';
```

Comment:
I changed this in a recent PR to parse it into a nice fraction /10 (I read that it is good to convert to scales that users are familiar with), the function to do this is sentimentScoreToNumerator(score) in script.js, could you use it here.

Reply:
Done thanks!